src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -64,22 +64,26 @@ class ArrayField extends React.Component {
     return schema.additionalItems;
   }
 
-  scrollToTop() {
-    setTimeout(() => {
-      // Hacky; won’t work if the array field is used in two pages and one isn’t
-      //  a BasicArrayField nor if the array field is used in three pages.
-      scroller.scrollTo(
-        `topOfTable_${this.props.path[this.props.path.length - 1]}${
-          this.isLocked() ? '_locked' : ''
-        }`,
-        window.Forms?.scroll || {
-          duration: 500,
-          delay: 0,
-          smooth: true,
-          offset: -60,
-        },
-      );
-    }, 100);
+  /**
+   * Scroll to an element, then focus on a button within the element
+   * @param {string} elementName - element with "name" attribute to scroll to
+   * @param {string} button - Button to focus within element
+   */
+  scrollToAndFocus(elementName, button = '') {
```

Comment:
What do you think about `scrollElementName` and `focusElementSelector` as argument names?

Reply:
Sounds good!